ui/src/components/executions/Executions.vue

```diff
@@ -104,11 +104,24 @@ export default {
             dailyReady: false,
         };
     },
+    mounted() {
+        //hack, but I did not found how to do this in a clean way
+        setTimeout(() => {
+            this.$el.querySelectorAll('thead>tr>th')[1].setAttribute("aria-sort", "descending")
+        },500)
+    },
     beforeCreate () {
-        const params = JSON.parse(localStorage.getItem('executionQueries') || '{}')
-        params.sort = 'state.startDate:desc'
-        params.status = this.$route.query.status || params.status || 'ALL'
-        localStorage.setItem('executionQueries', JSON.stringify(params))
+        const queries = JSON.parse(localStorage.getItem('executionQueries') || '{}')
+        queries.sort = 'state.startDate:desc'
```

Comment:
Why not use a simple 
```js
if (queries.sort === undefined) {
  queries.sort = 'state.startDate:desc'
}
```

Reply:
why not ,yes. I did this to prevent vue router route duplication. If I only compare sort field, we might have the duplicate route anyway.

Reply:
there is duplicate route everywhere ;) 
This will avoid the hack also I think

Reply:
We want to keep state in url as much as possible, and this is why the api ajax call is made from route. this enforces url as state for the app.

Reply:
so do we keep my solution, ok go with yours ?